docs/application/dotnet/guides/nui/relative-layout.md

```diff
@@ -0,0 +1,61 @@
+| `RightRelativeOffset`  | float           | Gets or sets the relative offset for right target. <br /> When the value is `0` the right edge of the child view is aligned to the left edge of the right target.<br/> When the value is `1` the right edges of the right target and child view are aligned.
+| `TopRelativeOffset`    | float           | Gets or sets the relative offset for top target. <br /> When the value is `0` the top edges of the top target and child view are aligned.<br/> When the value is `1` the top edge of the child view is aligned to the bottom edge of the top target.
+| `BottomRelativeOffset` | float           | Gets or sets the relative offset for bottom target. <br /> When the value is `0` the bottom edge of the child view is aligned to the top edge of the right target.<br/>When the value is `1` the bottom edges of the bottom target and child view are aligned.
+| `HorizontalAlignment`  | RelativeLayout.Alignment| Gets or sets the horizontal alignment of this child view.|
+| `VerticalAlignment`  | RelativeLayout.Alignment| Gets or sets the vertical alignment of this child view.|
+| `FillHorizontal`  | bool | Gets or sets the boolean value whether the child fills its horizontal space.|
+| `FillVertical`  | bool | Gets or sets the boolean value whether the child fills its vertical space.|
+
+<a name="targetandrelativeoffset"></a>
+## Target and RelativeOffset
+
+ The `Target` and the `RelativeOffset` properties determine the size of layout space.
+
+ The `Target` indicates the target view in which each edge of the layout space is aligned. The default value is `null`, which refers to the parent's relative layout.
+
+ The `RelativeOffset` moves each edge of layout space in proportion to the target view size.<br/>
+When the value is `0.0f`, the edge of the layout space is aligned with the left or top edge of the target view.<br/>
+When the value is `1.0f`,  the edge of the layout space is aligned with the right or bottom edge of the target view:
+
+![RelativeLayoutTargetRelativeOffset](./media/relativeLayoutTargetRelativeOffset.png)
+
+<a name="alignment"></a>
+## Alignment
+
+ The `HorizontalAlignment` and the `VerticalAlignment` properties describe how to align the child's view with the layout space. The default value is `Start` for both axes:
+ |Start|Center|End|
+ |:---:|:---:|:---:|
+ |![RelativeAlignStart](./media/relativeLayoutAlignStart.png)|![RelativeAlignCenter](./media/relativeLayoutAlignCenter.png)|![RelativeAlignEnd](./media/relativeLayoutAlignEnd.png)|
+ 
+<a name="fill"></a>
+## Fill
+
+ The `FillHorizontal` and the `FillVertical` properties are specified by boolean value and specifies whether to fill the layout space or not. The default is `false` for both axes:
+ |false|true|
+ |:---:|:---:|
+ |![RelativeAlignCenter](./media/relativeLayoutAlignCenter.png)|![RelativeFill](./media/relativeLayoutFill.png)|
+
+## Related Information
```

Comment:
Global: 
1. Please use sentence case for the headings.
2. No links in the page: not required to check.

Reply:
> 2. No links in the page: not required to check.

Sorry, i don't understand about `links`.  Could you please elaborate on comment `2.`?

Reply:
@dasg34 just a precheck. Thanks please ignore 2.